src/content-scripts/init-API.ts

```diff
@@ -0,0 +1,46 @@
+import store from './store/store';
+import axios from 'axios';
+import { setAxiosConfig } from 'redux-json-api';
+
+function getExtensionCookie(name: string) {
```

Comment:
Nice! I am gonna use it with ppGA soon

Reply:
I didn't export it just yet, but I thought of it as a util-to-be ;)